tutorials/example_keras_qat.py

```diff
@@ -0,0 +1,130 @@
+
+import tensorflow as tf
+from keras.datasets import mnist
+from keras import Model, layers, datasets
+import model_compression_toolkit as mct
+import numpy as np
+
+
+def get_tpc():
+    # Generate a TargetPlatformModel with power of two quantization with 3 bits for
+    # activations and 2 bits for weights
+    tp = mct.target_platform
```

Comment:
Please fix comment: We're generating a Keras TargetPlatformCapabilities with 3 bits...